tests/pytorch/nightly/dlrm.libsonnet

```diff
@@ -19,42 +19,48 @@ local tpus = import 'templates/tpus.libsonnet';
 local utils = import 'templates/utils.libsonnet';
 
 {
-  local command_common = |||
-    pip install onnx
-    git clone --recursive https://github.com/pytorch-tpu/examples.git
-    python examples/deps/dlrm/dlrm_tpu_runner.py \
-      --arch-sparse-feature-size=64 \
-      --arch-mlp-bot=512-512-64 \
-      --arch-mlp-top=1024-1024-1024-1 \
-      --arch-interaction-op=dot \
-      --lr-num-warmup-steps 10 \
-      --lr-decay-start-step 10 \
-      --num-batches=1000 \
-      --data-generation="random" \
-      --numpy-rand-seed=727 \
-      --print-freq 100 \
-      --num-indices-per-lookup=100 \
-      --use-tpu \
-      --metrics-debug \
-      --num-indices-per-lookup-fixed \
-  |||,
-  local dlrm = common.PyTorchTest {
-    modelName: 'dlrm',
+  local dlrm = {
```

Comment:
Can you change this line back to `local dlrm = common.PyTorchTest {` and remove `common.PyTorchTest + ` from the configs below? If all of the tests use `common.PyTorchTest` as a base (which should always be the case, except for TPU Node Pod tests), it's better to include it in `dlrm` instead.

Reply:
makes sense